lib/seahorn/BvOpSem2TrackingRawMemMgr.cc

```diff
@@ -113,9 +146,11 @@ TrackingRawMemManager::MemValTy TrackingRawMemManager::storeValueToMem(
   const unsigned byteSz =
       m_sem.getTD().getTypeStoreSize(const_cast<llvm::Type *>(&ty));
   ExprFactory &efac = ptr->efac();
-  // TODO: use zeroed memory on m_main, m_metadata instead of explicit
+  // TODO: use zeroed memory on m_main, m_w_metadata instead of explicit
   // init
   MemValTy res(m_ctx.alu().ui(0UL, wordSizeInBits()),
```

Comment:
I am confused about this initialization. This should only be used when `store` operation fails. Should it be just a null indicating that operation failed?